src/Utility/Schema.php

```diff
@@ -21,18 +21,18 @@
 class Schema
 {
     /**
-     * Return unique alphabetically ordered right types from schema $schema
+     * Return unique alphabetically ordered object types from map
      *
-     * @param array $schema Schema data.
+     * @param array $relationsSchema The relations schema
      * @return array
      */
-    public static function rightTypes(array $schema): array
```

Comment:
Since it is a `public` method I think it would be better to maintain this method and mark it as `@deprecated Will be dropped in 5.x version`.

Reply:
method restored as deprecated